sdks/go/pkg/beam/core/runtime/exec/translate.go

```diff
@@ -418,7 +418,8 @@ func (b *builder) makeLink(from string, id linkID) (Node, error) {
 					}
 					u = n
 					if urn == urnProcessSizedElementsAndRestrictions {
-						u = &ProcessSizedElementsAndRestrictions{PDo: n}
+						transform.GetInputs()
```

Comment:
Copy Pasta?

Reply:
Something like that. Definitely not supposed to be there. Removed.